package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@arthurgeron/eslint-plugin-react-usememo",
-  "version": "2.4.4",
+  "version": "2.5.0",
```

Comment:
Shouldn't this be a v3? As it's supporting flat config with eslint v9?

Reply:
We will continue to support the older ESLint V8 configuration style. Otherwise, this would be a major version increment.